### PR TITLE
Katib updates

### PR DESCRIPTION
**Pull request by MaxKavun**
**Which issue is resolved by this Pull Request:**
Resolves #

**Description of your changes:**
user roles update and values fixes

**Checklist:**
- [ ] Unit tests pass:
  **Make sure you have installed kustomize == 5.2.1+**
    1. `make generate-changed-only`
    2. `make test`

### CODE REVIEW

File: charts/kubeflow/templates/katib/user-roles/clusterrole.kubeflow-katib-admin.yaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: kubeflow-katib-admin
   labels:
-    rbac.authorization.kubeflow.org/aggregate-to-kubeflow-admin: "true"
+    {{- include "kubeflow.kubeflowRoles.kubeflowAdminRoleLabel" . | nindent 4 }}
+  name: {{ include "kubeflow.katib.kfNbAdminClusterRoleName" . }}
```
Comment (kromanow94) [inline charts/kubeflow/templates/katib/user-roles/clusterrole.kubeflow-katib-admin.yaml]: Maybe let's rename to `kubeflow.katib.adminClusterRoleName`.

**Comment by kromanow94** [comment]
@MaxKavun can you investigate this error? I got it using the `quickstart.helm.local.sh` script:
```
Error: 1 error occurred:
        * ClusterRole in version "v1" cannot be handled as a ClusterRole: json: cannot unmarshal string into Go struct field LabelSelector.aggregationRule.clusterRoleSelectors.matchLabels of type map[string]string
```

File: charts/kubeflow/templates/katib/user-roles/clusterrole.kubeflow-katib-admin.yaml (hunk 1 above)
Comment (MaxKavun) [reply charts/kubeflow/templates/katib/user-roles/clusterrole.kubeflow-katib-admin.yaml]: Updated

**Comment by MaxKavun** [comment]
@kromanow94 that was tested and fixed

**Pull request merged**